server/package.json

```diff
@@ -8,17 +8,21 @@
     "lint": "eslint --ignore-path ../.gitignore .",
     "start": "node index.js",
     "start:debug": "nodemon --inspect index.js",
-    "test": "echo 'test me bby uwu'"
+    "test": "mocha **/*.test.js"
   },
   "dependencies": {
     "body-parser": "^1.19.0",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
+    "express-graphql": "^0.9.0",
+    "graphql": "^14.6.0",
     "mongodb": "^3.5.5",
     "node-github-graphql": "^0.2.7",
     "ws": "^7.2.3"
   },
   "devDependencies": {
-    "nodemon": "^2.0.2"
```

Comment:
Sort of unrelated thought: should we add an engine for the server? Node 12+?

Reply:
I don't usually think about the engines property because I personally always want to work against the latest version, but if you want me to add an engines property I can. I know it can sometimes be necessary for some CD integrations, but I'm not very familiar with that part of the project _yet_.

Reply:
I usually set an engine for a min node version (just in case we're using features that only exist in specific versions). Are we doing anything special?  Doesn't look like it

Reply:
Oh, actually the current engines property is preventing me from using latest. 😆 

It is set to 12 in the root package.json but I'm on 13 locally, requiring me to change my version anytime I work on this project in a new session. I forgot about it until a restart put me back on latest and I tried yarn'ing.

I'll update it so I can actually use latest. 👌

Reply:
https://github.com/Robdel12/weatherstation/pull/226/commits/5f7e173115badd77f45db21a780d2cded60618bb